### ISSUE TITLE
[Feature Request] Add option to grow reversed order status effects from right to left

### ISSUE THREAD

**Issue by ryankhart** [Caraxi/PartyListLayout#23]
My reason for downloading this plugin was to have my party list on the far right side of my screen and have the status effects of party members expand leftward out of the left side of the party list since most of the time, that status effect area of usually empty anyway. I was disappointed to find out that, even though this plugin did let me reverse the order of the status effects, it still seemed to grow from left to right. I want it to grow from right to left.

**Comment by Caraxi** [comment]
set offset to negative and it will grow right to left

**Comment by ryankhart** [comment]
Whoa! That worked! And you were fast to respond. I'm going to close this issue now. Hopefully this closed issue will help other users who might be about to post about this.

I think, when I saw that there was an option for reversed order, that closed off a part of my mind that there was no other place to search for if that option didn't work or another option clearly labelled as a grow right to left wasn't visible as it is in DelvUI.

I also use DelvUI, but I just couldn't get used to DelvUI's implementation of the Party List the last time I tried.

Thanks for this cool plugin. Now I can make some more efficient use of screen space.
